ietf/templates/meeting/proceedings_activity_report.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+{% load ams_filters ietf_filters cache %}
+{% block title %}IETF {{ meeting.number }} Proceedings - Activity Report{% endblock %}
+{% block content %}
+    {% cache 3600 proceedings_activity_report meeting.number %}
```

Comment:
I suspect this is wrong enough to need a change.
Instead of varying on meeting.number, you probably need to vary cach on sdate and edate.
Please check other places that the templates or views use the cache.

Reply:
For this template, proceedings_activity_report, meeting.number should be correct. For the monthly report, templates/iesg/ietf_activity_report.html, caching isn't installed, but it could be added, and should use sdate.